pkg/cloud/client_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloud_test
+
+// import (
```

Comment:
Why don't you remove this file as this file is technically empty?

Reply:
Whoops. Didn't mean for it to be commented out like that.

Was trying to figure out why the suite was hanging. It's fixed now.